pkg/vault/test/foundry/CompositeLiquidityRouterNestedPools.t.sol

```diff
@@ -40,30 +44,68 @@ contract CompositeLiquidityRouterNestedPoolsTest is BaseVaultTest {
             "parentPool"
         );
 
-        vm.startPrank(lp);
-        uint256 childPoolABptOut = _initPool(childPoolA, [poolInitAmount, poolInitAmount].toMemoryArray(), 0);
-        uint256 childPoolBBptOut = _initPool(childPoolB, [poolInitAmount, poolInitAmount].toMemoryArray(), 0);
-
-        uint256[] memory tokenIndexes = getSortedIndexes([childPoolA, childPoolB, address(dai)].toMemoryArray());
-        uint256 poolAIdx = tokenIndexes[0];
-        uint256 poolBIdx = tokenIndexes[1];
-        uint256 daiIdx = tokenIndexes[2];
-
-        uint256[] memory amountsInParentPool = new uint256[](3);
-        amountsInParentPool[daiIdx] = poolInitAmount;
-        amountsInParentPool[poolAIdx] = childPoolABptOut;
-        amountsInParentPool[poolBIdx] = childPoolBBptOut;
-        vm.stopPrank();
+        childPoolERC4626 = _createPool([address(waDAI), address(weth)].toMemoryArray(), "childPoolERC4626");
+        parentPoolWithoutWrapper = _createPool(
+            [address(childPoolERC4626), address(usdc)].toMemoryArray(),
+            "parentPoolWithoutWrapper"
+        );
```

Comment:
Ok, this is the closest to `bb-a-usd / vanilla` and the case that we want to support the most :+1: